Dangerfile

```diff
@@ -1,5 +1,5 @@
 def fail_if_no_supported_label_found
-  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test"]
+  supported_types = ["breaking", "build", "ci", "docs", "feat", "fix", "perf", "refactor", "style", "test", "next_release"]
```

Comment:
Hmm I wonder if we should rename to `release`... I'm guessing there will be 2 PRs usually tagged with this, the "bump" PR which is the actual release, and the "prepare next version" PR, bumping to the next snapshot.

Reply:
I don't have a strong preference, I went with `next_release` because that's a label that already exists (and I think it's default in GitHub?)

Reply:
Yeah me neither... I just didn't know if the "next" was necessary... But don't have a strong opinion so we can leave as is